main.lua

```diff
@@ -127,6 +137,58 @@ local function load_twitch_chat(is_new_session)
             msg_separator = ""
         end
 
+        if not o.break_long_lines then
+            goto skip_line_break
+        end
+
+        local msg_body
+        if o.show_name then
+            msg_body = msg_part_2
+        else
+            msg_body = msg_part_1
+        end
```

Comment:
A better way is move the whole block down a bit and just use `msg_line`. Username can be long too.